flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/JobExceptionsInfo.java

```diff
@@ -81,6 +81,22 @@ public int hashCode() {
 		return Objects.hash(rootException, rootTimestamp, allExceptions, truncated);
 	}
 
+	public String getRootException() {
+		return rootException;
+	}
+
+	public Long getRootTimestamp() {
+		return rootTimestamp;
```

Comment:
getters should be annotated with `@JsonIgnore` to prevent wacky behaviors if the java/json field names differ.